test/test_reporter.rb

```diff
@@ -20,7 +20,7 @@ def test_failing_suite
     output = exec_reporter(build_id: build_id)
 
     assert_match "Failed examples", output
-    assert_match "bin/rspec --seed 1234 ./spec/fail_1_spec.rb:3", output
+    assert_match "bin/rspec ./spec/fail_1_spec.rb:3", output
```

Comment:
Why did the seed was removed here? It's still makes sense to provide it, since it will cause the examples to run in the same order, which in turn makes reproducing failures easier.